Reflections/ReflectionWeek4.md

```diff
@@ -0,0 +1,17 @@
+1. For this week I didn't accomplish that much as I have been busy with midterms.
+- I was able to get a working screen color changing slider so the user can actually make the design of their app be whatever they want because I wanted the color wheel to represent the different colors the app can be.
+- I didn't get finished but I recently started using the free to use editing application because I didn't have the money for Adobe After Effects. I used this to begin creating the animation for the logo when the app starts up.
+- I didn't get to implement the search filter yet because I didn't fully understand how Yelp did it but it was I who was getting mixed up.
+
+2. The biggest challenge that I faced this week was trying to stay on task while having 2 other midterms to study for, so I didn't get as much done as I wanted to. I am going to try and put it into overdrive this weekend and work extra hard.

```

Comment:
Hang in there! it  was one week and midterms are over so you can definitely ca get back on track this week!